docs/01-infrastructure-gcp.md

```diff
@@ -52,29 +57,50 @@ gcloud compute networks subnets create kubernetes \
   --region us-central1
 ```
 
+While your virtual network exists across all GCP regions, a subnet is a range of private IP addresses within a single region. Instances are created within a subnet.
+
+`10.240.0.0/24` means IPs from `10.240.0.0` to `10.240.0.254`.
+
 ### Create Firewall Rules
 
+https://cloud.google.com/compute/docs/vpc/firewalls
+
+A GCP network also acts as a firewall. By default no connections are allowed from the outside world, and connections between instances are also forbidden. We can add firewall rules to allow our instances to talk to each other within the network.
+
+This creates a rule called `allow-internal` which allows TCP, UDP and ICMP connections between all machines in your `10.240.0.0/24` subnet, and also enables those machines to talk to the CIDR range `10.200.0.0/16`:
```

Comment:
I have to admit I don't understand the need for the 10.200.0.0/16 range - I managed to figure out it was something to do with the acronym CIDR but I don't understand why it's needed or what effect it has.

Reply:
@simonw The 10.200.0.0/16 range is the set of IP's from which your pods will get their IP addresses. 
This range is passed as a parameter to kubernetes-controller-manager https://github.com/kelseyhightower/kubernetes-the-hard-way/blob/master/docs/05-kubernetes-controller.md

More details on controller manager flags : https://kubernetes.io/docs/admin/kube-controller-manager/

Reply:
Fantastic! Thanks @harshal-shah that's exactly what I needed to know.